pyproject.toml

```diff
@@ -176,7 +176,7 @@ markers =[
 ]
 log_level = "info"
 filterwarnings = [
-    #"error",  # Turn all warnings not explicitly filtered below into errors
+    "error",  # Turn all warnings not explicitly filtered below into errors
```

Comment:
This should have been uncommented in this PR
 - https://github.com/great-expectations/cloud/pull/54

Reply:
cc @Shinnnyshinshin

Reply:
Resolved 
- https://github.com/great-expectations/cloud/pull/88